ocaml/idl/datamodel_repository.ml

```diff
@@ -96,6 +96,38 @@ let apply =
     ~allowed_roles:(_R_POOL_OP ++ _R_CLIENT_CERT)
     ()
 
+let apply_livepatch =
+  call ~name:"apply_livepatch" ~in_oss_since:None ~lifecycle:[]
+    ~doc:"Apply a live patch on a host"
```

Comment:
Inconsistent spelling: _livepatch_ and _live patch_ are both being used. Grammar: _Apply ... patch to a host_